deepchem/models/torch_models/cgcnn.py

```diff
@@ -0,0 +1,255 @@
+import torch
+import numpy as np
+import torch.nn as nn
+import torch.nn.functional as F
+
+from deepchem.models.torch_models.torch_model import TorchModel
+
+
+class CGCNNLayer(nn.Module):
+  """The convolutional layer of CGCNN.
+
+  This class was implemented using DGLGraph methods.
+  Please confirm how to use DGLGraph methods from below link.
+  See: https://docs.dgl.ai/en/0.4.x/tutorials/models/1_gnn/9_gat.html
```

Comment:
Could you add a usage example to the docstring?

Reply:
I added!